ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -9,46 +9,56 @@
 {-# LANGUAGE RecordWildCards            #-}
 {-# LANGUAGE ScopedTypeVariables        #-}
 
-module Ouroboros.Consensus.Storage.LedgerDB.HD.LMDB
-  ( newLMDBBackingStore
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# HLINT ignore "Use <$>" #-}
```

Comment:
These two should go away before merging. In any case, one can use
```{- HLINT ignore "Use <$>" -}```
and it should work without being a Pragma